packages/terra-notification-dialog/src/NotificationDialog.module.scss

```
@@ -42,98 +33,120 @@
   }
 
-  .icon-container {
-    flex-shrink: 0; // This will keep text-wrapper from overlapping icon-container in IE11
-    
-    > .icon {
-      display: inline-block;
-      height: var(--terra-notification-dialog-icon-height, 2rem);
-      margin-right: var(--terra-notification-dialog-icon-margin-right, 0.57142rem);
-      width: var(--terra-notification-dialog-icon-width, 2rem);
-    }
+  .warning {
+    background-image: var(--terra-notification-dialog-warning-background-image, linear-gradient(to bottom, #fff3ce, #fff));
+    border-top: var(--terra-notification-dialog-warning-border-top, 7px solid #ffc20a);
+  }
 
-    > .alert {
-      background: var(--terra-notification-dialog-alert-icon-background, inline-svg("<svg viewBox='0 0 49 49' xmlns='http://www.w3.org/2000/svg'><path fill='#e50000' d='M1.2 45c-1.1 0-1.6-.8-1-1.7L23 3.7c.5-1 1.4-1 2 0l22.8 39.6c.5 1 .1 1.7-1 1.7H1.2z'/><path fill='#fff' d='M21.5 36.7h5V42h-5v-5.3zm0-22.7h5v17.3h-5V14z'/></svg>"));
-    }
+  .info {
+    background-image: var(--terra-notification-dialog-info-background-image, linear-gradient(to bottom, #e4e2e2, #fff));
+    border-top: var(--terra-notification-dialog-info-border-top, 7px solid #c8cacb);
+  }
 
-    > .warning {
-      background: var(--terra-notification-dialog-warning-icon-background, inline-svg("<svg viewBox='0 0 49 49' xmlns='http://www.w3.org/2000/svg'><path fill='#ffc20a' d='M1.2 45c-1.1 0-1.6-.8-1-1.7L23 3.7c.5-1 1.4-1 2 0l22.8 39.6c.5 1 .1 1.7-1 1.7H1.2z'/><path fill='#1c1f21' d='M47.7 43.3L25 3.7c-.3-.5-.6-.7-1-.7s-.7.2-1 .7L.2 43.3c-.5 1-.1 1.7 1 1.7h45.5c1.1 0 1.6-.8 1-1.7zm-45.9.2L24 5l22.2 38.5H1.8zm19.7-6.8h5V42h-5v-5.3zm0-22.7h5v17.3h-5V14z'/></svg>"));
-    }
+  .custom {
+    background-image: var(--terra-notification-dialog-custom-background-image, linear-gradient(to bottom, #e4e2e2, #fff));
+    border-top: var(--terra-notification-dialog-custom-border-top, 7px solid #c8cacb);
+  }
 
-    > .info {
-      background: var(--terra-notification-dialog-info-icon-background, inline-svg("<svg viewBox='0 0 49 49' xmlns='http://www.w3.org/2000/svg'><path fill='#007cc3' d='M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z'/><path fill='#fff' d='M24.1 32.7c.7-1.8 6.3-11 2.4-13.5-2.2-1.4-5.2-.5-7.6 1.2l-.2 1.6c1.4-.5 2.8-.5 3.2.7 1 2.9-2.5 8-3.5 10.7-2.6 7.2 3.7 8.1 8.1 5.1l.2-1.7c-2.3.6-4-.3-2.6-4.1zM26.5 8c1.9 0 3.5 1.6 3.5 3.5S28.4 15 26.5 15 23 13.4 23 11.5 24.6 8 26.5 8z'/></svg>"));
-    }
+  .header {
+    flex: 0 0 auto;
+    padding: var(--terra-notification-dialog-header-padding, 0 1.42857rem); // left and right padding
+    z-index: 1;
+  }
+  
+  .header-content {
+    border-bottom: var(--terra-notification-dialog-header-content-border-bottom, 1px solid #dedfe0);
+    display: flex;
+    flex-direction: row;
+    padding: var(--terra-notification-dialog-header-content-padding, 1.35714rem 0 0.5rem); // top and bottom padding to squish border bottom
```

reduce top padding by 2px to help icon align with titles
```suggestion
    padding: var(--terra-notification-dialog-header-content-padding, 1.21428rem 0 0.5rem); // top and bottom padding to squish border bottom
```

---

updated: https://github.com/cerner/terra-framework/pull/1105/commits/b06b04c4f64702b00a0f5e2e4159aadcd23bd23c#diff-199a5a3214c6de6b926614643935d3a5R78